www/content/server-examples.md

```diff
@@ -200,6 +200,10 @@ These examples may make it a bit easier to get started using htmx with your plat
 
 ## Go
 
+### htmgo
+
+- <https://htmgo.dev>
+
 ### templ
 
 - <https://templ.guide/server-side-rendering/htmx>
```

Comment:
Do you mind moving it to the bottom of the list, I try to keep things in first-come-first-serve order to avoid fights over that.

Reply:
Yeah np, pushed update